src/Diagnostics.ModelsAndUtils/Models/ResponseExtensions/SummaryCard.cs

```diff
@@ -0,0 +1,85 @@
+using System;
+using System.Collections.Generic;
+using System.Data;
+using System.Linq;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Converters;
+
+namespace Diagnostics.ModelsAndUtils.Models.ResponseExtensions
+{
+    public class SummaryCard
+    {
+        public string Title { set; get; }
+
+        public string Message { set; get; }
+
+        [JsonConverter(typeof(StringEnumConverter))]
+        public SummaryCardStatus Status { set; get; }
+
+        public string Description { set; get; }
+
+        public string DetectorLink { set; get; }
+
+        public SummaryCard(SummaryCardStatus status, string title,string message,string description,string detectorLink) 
+        {
+            this.Status = status;
+            this.Message = message;
+            this.Description = description;
+            this.Title = title;
+            this.DetectorLink = detectorLink;
+        }
+    }
+
+    public enum SummaryCardStatus
+    {
+        Critical,
+        Warning,
+        Info,
+        Success,
+        None
+    }
+
+    public static class ResponseSummaryCardExtension
```

Comment:
Could you add some example code how to use the new rendering in the comments like other rendering type?

Reply:
Yes, I can write an example detector like **[Card View](https://applens.azurewebsites.net/subscriptions/6bbc1a17-0990-42cb-97c6-1cfc9f92472b/resourceGroups/GenyaWebApp/providers/Microsoft.Web/sites/Genya1/detectors/webappdown2?startTime=2019-11-23T23:53&endTime=2019-11-24T23:37)** detector to show how to use this new rendering type

Reply:
Yes, I can write an new detector(looks like existing [Card View](https://applens.azurewebsites.net/subscriptions/6bbc1a17-0990-42cb-97c6-1cfc9f92472b/resourceGroups/GenyaWebApp/providers/Microsoft.Web/sites/Genya1/detectors/webappdown2?startTime=2019-11-23T23:53&endTime=2019-11-24T23:37)) to show how to use this new render type.